tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion_test.cc

```diff
@@ -593,14 +589,10 @@ TEST_F(HorizontalLoopFusionTest, TraversalOrder) {
   // know all the sqrt instructions are fused into a kernel. Note that if we
   // traverse from def-to-use (i.e., top-to-down) instead of use-to-def, we
   // will end up having 3 fusions instead of 2.
-  size_t total_fusion_instrs = 0;
-  for (const HloInstruction* instr :
-       module->entry_computation()->instructions()) {
-    if (instr->opcode() == HloOpcode::kFusion) {
-      ++total_fusion_instrs;
-    }
-  }
-  EXPECT_EQ(total_fusion_instrs, 2);
+  auto is_fusion = [](const HloInstruction* instr) {
+                     return instr->opcode() == HloOpcode::kFusion;
+                   };
+  EXPECT_EQ(absl::c_count_if(module->entry_computation()->instructions(), is_fusion), 2);
```

Comment:
Why not inline the lambda? Or go even further and make this a helper reusable method of this test?

Reply:
done.